addons/mute-project/addon.json

```diff
@@ -7,6 +7,13 @@
       "link": "https://scratch.mit.edu/users/TheColaber"
     }
   ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "On macOS, you can also use the Cmd key.",
```

Comment:
We usually use `On macOS, use the \"Cmd\" key instead of the \"Ctrl\" key.`

Reply:
changed